### PR TITLE
Create scripts of packer images

### PR DESCRIPTION
**Pull request by xenirio**

### CODE REVIEW

File: packer/scripts/install-opx.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -x
```
Comment (vshatravenko) [inline packer/scripts/install-opx.sh:1]: Let's rename this to `install-opendax.sh` since opx is a already used

File: .drone.yml [hunk 2]
```diff
@@ -29,3 +30,59 @@ steps:
...
+    PROJECT_ID:
+      from_secret: project_id
+  commands:
+    - packer build -var project_id=$PROJECT_ID ./packer/gcp-packer.json
+
+trigger:
+  event:
+    - push
+  branch:
+    - feature/packer-ci
```
Comment (vshatravenko) [inline .drone.yml:88]: Let's put master here before merge

File: .drone.yml [hunk 3]
```diff
@@ -29,3 +30,59 @@ steps:
...
+    - packer build -var project_id=$PROJECT_ID ./packer/gcp-image-base.json
+
+- name: Build GKE Opendax image
+  image: hashicorp/packer
+  environment:
+    GOOGLE_APPLICATION_CREDENTIALS: cred.json
+    PROJECT_ID:
+      from_secret: project_id
+  commands:
+    - packer build -var project_id=$PROJECT_ID ./packer/gcp-packer.json
```
Comment (vshatravenko) [inline .drone.yml:82]: Let's try to build all of them in parallel pipelines like here https://github.com/openware/peatio/blob/master/.drone.yml

File: packer/aws-packer.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
+{
+  "variables": {
+    "region": "eu-central-1",
```
Comment (vshatravenko) [inline packer/aws-packer.json:3]: This should be configurable with a default value

File: packer/do-packer.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
+{
+  "variables": {
+    "api_token": "{{ env `DIGITALOCEAN_API_TOKEN` }}",
+    "region": "fra1"
```
Comment (vshatravenko) [inline packer/do-packer.json:4]: This should be configurable with a default

File: packer/gcp-image-base.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
+{
+  "variables": {
+    "account_file": "{{ env `GOOGLE_APPLICATION_CREDENTIALS` }}",
+    "project_id": "",
+    "zone": "europe-west1-b"
```
Comment (vshatravenko) [inline packer/gcp-image-base.json:5]: This should be configurable with a default

File: packer/gcp-packer.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
+  "variables": {
+    "account_file": "{{ env `GOOGLE_APPLICATION_CREDENTIALS` }}",
+    "project_id": "",
+    "zone": "europe-west1-b"
```
Comment (vshatravenko) [inline packer/gcp-packer.json:5]: This should be configurable with a default

File: packer/scripts/install-ruby-gcp.sh [hunk 8]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
+
+sudo -u deploy bash <<EOS
+  gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+  curl -sSL https://get.rvm.io | bash -s stable
+
+  source /home/deploy/.rvm/scripts/rvm
+  cd /home/deploy
+  rvm install --quiet-curl 2.6.1
```
Comment (vshatravenko) [inline packer/scripts/install-ruby-gcp.sh:9]: We should try to have the version updated here

**Review by vshatravenko — CHANGES_REQUESTED** [review]

File: packer/scripts/install-ruby-gcp.sh (hunk 8 above)
Comment (xenirio) [reply packer/scripts/install-ruby-gcp.sh:9]: For the ruby version update, I take the ruby version from `.ruby-version` file in `opendax`

**Review by vshatravenko — APPROVED** [review]

**Pull request merged**